StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/STPAPIClient+Link.swift

```diff
@@ -110,11 +112,13 @@ extension STPAPIClient {
             }
 
             post(
-                resource: useModernMobileEndpoints ? modernEndpoint : legacyEndpoint,
+                resource: useMobileEndpoints ? modernEndpoint : legacyEndpoint,
```

Comment:
nit: maybe we should update modern endpoint to `mobileEndpoint` (At this point modernEndpoint looks like a typo)